File: mobileAppClient/mobileAppClient/Models/HistoryItem.cs
using System;
namespace mobileAppClient
{
    /*
     * Class to handle history items coming from the server with all relevant details.
     */ 
    public class HistoryItem
    {
        public string action { get; set; }
        public string description { get; set; }
        public CustomDateTime dateTime { get; set; }
        public int id { get; set; }

    }
}

File: desktopClient/src/main/java/seng302/User/HistoryItem.java
package seng302.User;

import java.time.LocalDate;
import java.time.LocalDateTime;

/**
 * Class for handling all history details of a User
 */
public class HistoryItem {
    private LocalDateTime dateTime;
    private String action;
    private String description;

    public HistoryItem(LocalDateTime dateTime, String action, String description) {
        this.dateTime = dateTime;
        this.action = action;
        this.description = description;
    }

    public HistoryItem(String action, String description) {
        this.dateTime = LocalDateTime.now();
        this.action = action;
        this.description = description;
    }

    public LocalDate getDate() {
        return dateTime.toLocalDate();
    }

    public LocalDateTime getDateTime() {
        return dateTime;
    }

    public String getAction() {
        return action;
    }

    public String getDescription() {
        return description;
    }
}
